source/include/Variants.h

```diff
@@ -60,7 +60,7 @@ class VariantAttributeManager {
     }
 };
 
-enum { kVariantArgErr = 1, kVariantIncompatibleType = 91};
+enum { kVariantArgErr = 1, kVariantIncompatibleType = 91, kVariantTypeMismatch = 1558 };
```

Comment:
What situations is error 1558 normally used for (since this is a stopgap case)?

Reply:
@spathiwa, good question. This is the closest thing we were able to find to indicate unsupported behavior. What we basically need is an error code indicating that some behavior is not supported in the target. Are you aware of an error code we can use instead?

Reply:
The error usually used for functionality not supported on a particular target (for example, desktop features not supported in RT, or compiler or scripting property nodes in a built app), is mgNotSupported, error code 53.
I think this would be a good choice here.

Reply:
Options:
1. 1558 - There is a type mismatch between the wire type and the internal data type of the reference. This problem might occur due to a type cast error.
2. 1057 - Type mismatch: Object cannot be cast to the specified type.
3. 2304 - The command is not supported on the target.
4. 1429 - Command not supported by Target (sigh, why do we need this AND the previous one!)
5. 53 - Manager call not supported.

@spathiwa The unsupported on current target would be the best fit, but is 53 the right code for the error for that? Explain error doesn't seem to match that description. 

Looking at the descriptions above, 1057 seems generic enough for this case. The others have too much detail that doesn't match what is actually going on.

Reply:
Talked to @cglzaguilar and we agreed that 2304 suits this case the best. Let me know if you disagree. I am going to update my tests and code with this error code otherwise.

Reply:
Actually, I think that one is currently only used for RT targets, so it's a little weird to use it here.

Reply:
Maybe we should consider a 'not implemented' error code, rather than 'not supported', since this is a temporary failing that will be addressed in the future.
There is Error 1072, 'This property or method is not implemented', but that's unfortunately a little too specific, since this isn't for a property or method.
Other existing errors expressing the concept of not implemented are either not in the LabVIEW core range (they are negative or large hex values), or they are specific to an add-on product (16502, for SMTP part of internet toolkit).

Maybe you should consider adding a new error code?
https://niweb.natinst.com/confluence/display/RDITAppsHelp/Using+the+NI+Error+Codes+db

Reply:
I would've added a new code if not for this PR blocking some committed work that is due this week. I am also hesitant because we will likely implement the actual feature and remove the error before ever releasing.

Although 2304 is currently only used by RT targets, there's nothing about its messaging that prevents us from using it, right? It is the target that is preventing this case from working (it works on Desktop and in CG).

Reply:
Ok. If it’s short/term, I’m fine with 2304.